pkg/apiserver/rest/apis/v1/types.go

```diff
@@ -483,6 +483,7 @@ type CreateProjectRequest struct {
 	Name        string `json:"name" validate:"checkname"`
 	Alias       string `json:"alias" validate:"checkalias" optional:"true"`
 	Description string `json:"description" optional:"true"`
+	Namespace   string `json:"namespace" optional:"true"`
```

Comment:
This can be optional I think, if the namespace is empty, just use the project name.

Reply:
yes